stable/overwrite/.travis.yml

```diff
@@ -1,17 +1,16 @@
 language: go
 
 go:
-  - 1.7
+  - 1.8.x
```

Comment:
why not `1.8` here?

Reply:
Dunno, I just copied and pasted from here:
https://docs.travis-ci.com/user/languages/go/#Specifying-a-Go-version-to-use